src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -11,25 +13,15 @@
 FLOWS_DIR = "./tests/test_configs/flows/print_input_flow"
 
 
+def is_matching(str_a, str_b):
```

Comment:
I am trying to match strings instead of lines.
For example:
The string **"promptflow.executor.flow_validator, line 311, raise InvalidAggregationInput"** will become **"promptflow.executor.flow_validator, , raise InvalidAggregationInput"**
The string **"promptflow.executor.flow_validator, line 312, raise InvalidAggregationInput"** will become **"promptflow.executor.flow_validator, , raise InvalidAggregationInput"**

So **"promptflow.executor.flow_validator, line 311, raise InvalidAggregationInput"** match **"promptflow.executor.flow_validator, line 312, raise InvalidAggregationInput"**

Reply:
rename it.